### PR TITLE
Return channel when waiting for shutdown

### PR DESCRIPTION
**Pull request by gregarndt**

### CODE REVIEW

File: runtime/shutdownmgr.go [hunk 1]
```diff
@@ -25,22 +29,24 @@ func NewShutdownManager(host string) ShutdownManager {
 
 // AWSShutdownManager is a ShutdownManager that will listen for shutdowns on the notification
 // api provided by AWS.
-type AWSShutdownManager struct{}
+type AWSShutdownManager struct {
```
Comment (jonasfj) [inline runtime/shutdownmgr.go]: Why do you export this type?

File: runtime/shutdownmgr.go [hunk 2]
```diff
@@ -25,22 +29,24 @@ func NewShutdownManager(host string) ShutdownManager {
...
-	c := make(chan struct{})
-	return c
+func (a *AWSShutdownManager) WaitForShutdown() <-chan struct{} {
+	return a.sc
 }
 
 // LocalShutdownManager simple ShutdownManager that could listen for shutdown events
 // suitable for a local non-cloud environment (such as SIGTERM).
-type LocalShutdownManager struct{}
+type LocalShutdownManager struct {
```
Comment (jonasfj) [inline runtime/shutdownmgr.go]: Same for this one...

Comment (gregarndt) [inline runtime/shutdownmgr.go]: Both have been changed

**Comment by gregarndt** [comment]
merge if I addressed your comments

**Pull request merged**